experimental/generation/generator/packages/library/src/substitutions.ts

```diff
@@ -152,13 +166,13 @@ function substitutions(path: string, bindings: any, copies?: number, seed?: stri
     return result.join(os.EOL)
 }
 
-export let SubstitutionsEvaluator = new expr.ExpressionEvaluator('substitutions',
+export const SubstitutionsEvaluator = new expr.ExpressionEvaluator('substitutions',
     expr.FunctionUtils.apply(
         args => {
-            let path = args[0]
-            let bindings = args[1]
-            let replications = args.length > 2 ? args[2] : undefined
-            let seed = args.length > 3 ? args[3] : undefined
+            const path = args[0]
+            const bindings = args[1]
+            const replications = args.length > 2 ? args[2] : undefined
+            const seed = args.length > 3 ? args[3] : undefined
```

Comment:
Up to you, I tend to prefer writing things like this as:

```typescript
expr.FunctionUtils.apply(
  ([path, bindings, replications, seed]) => {
    ...
  }
)
```

It's rarely necessary to explicitly opt for a default value of `undefined`.

```
> [][1]
< undefined
```